File: Quizify-FrontEnd/src/app/quizify/classes/question.ts
import { Category } from './category';
import { Topic } from './topic';
import { Admin } from './admin';

export class Question {
    id: string;
    category: Category;
    topic: Topic;
    level: string;
    type: string;
    statement: string;
    options: string[];
    answer: string;
    admin: Admin;
}

File: search-service/src/main/java/com/stackroute/quizify/kafka/domain/Question.java
package com.stackroute.quizify.kafka.domain;

import lombok.Data;
import org.springframework.data.annotation.Id;

import java.util.List;

@Data
public class Question {
    @Id
    private long id;
    private Category category;
    private Topic topic;
    private String tag;
    private String genre;
    private String level;
    private String type;
    private String statement;
    private List<String> options;
    private String answer;
    private String playerAnswer;
    private String timeStamp;
    private Admin admin;
}
